docs/user/integration/dynatrace/README.md

```diff
@@ -137,12 +137,18 @@ To start ingesting custom spans and Istio spans, you must enable the Istio traci
         output:
             otlp:
                 endpoint:
-                    value: https://{ENVIRONMENT_ID}.live.dynatrace.com/api/v2/otlp
+                    valueFrom:
+                        secretKeyRef:
+                            name: dynakube
+                            namespace: ${DYNATRACE_NS}
+                            key: apiurl
+                path: v2/otlp/v1/metrics
                 headers:
                     - name: Authorization
+                      prefix: Api-Token
                       valueFrom:
                           secretKeyRef:
-                              name: dynatrace-token
+                              name: dynakube
                               namespace: ${DYNATRACE_NS}
                               key: apiToken
```

Comment:
The apiToken is the one used by the oneAgent usually and does not cover the required scopes